Server/resources/accounts/duelyst.account.json

```diff
@@ -1 +1,8202 @@
_anything","action":{"hpChange":-8,"durable":true,"duration":1},"target":{"dimensions":{"row":1,"column":1},"owner":{"own":true,"enemy":true},"cardType":{"hero":true,"minion":true,"player":true},"attackType":{"melee":true,"ranged":true,"hybrid":true}},"availabilityType":{"onPut":true}}],"manaCost":8,"price":100,"remainingNumber":20},{"name":"Spiral Technique","description":"Deal 8 damage to anything","cardId":"duelyst_SpiralTechnique_2","spriteName":"icon_f2_spiraltechnique","type":"SPELL","spells":[{"spellId":"deal_8_to_anything","action":{"hpChange":-8,"durable":true,"duration":1},"target":{"dimensions":{"row":1,"column":1},"owner":{"own":true,"enemy":true},"cardType":{"hero":true,"minion":true,"player":true},"attackType":{"melee":true,"ranged":true,"hybrid":true}},"availabilityType":{"onPut":true}}],"manaCost":8,"price":100,"remainingNumber":20},{"name":"Spiral Technique","description":"Deal 8 damage to anything","cardId":"duelyst_SpiralTechnique_3","spriteName":"icon_f2_spiraltechnique","type":"SPELL","spells":[{"spellId":"deal_8_to_anything","action":{"hpChange":-8,"durable":true,"duration":1},"target":{"dimensions":{"row":1,"column":1},"owner":{"own":true,"enemy":true},"cardType":{"hero":true,"minion":true,"player":true},"attackType":{"melee":true,"ranged":true,"hybrid":true}},"availabilityType":{"onPut":true}}],"manaCost":8,"price":100,"remainingNumber":20},{"name":"Tempest","description":"Deal 2 Damage to ALL characters","cardId":"duelyst_Tempest_1","spriteName":"icon_f1_tempest","type":"SPELL","spells":[{"spellId":"deal_2_dmg_everything","action":{"hpChange":-2,"durable":true,"duration":1},"target":{"dimensions":{"row":10,"column":18},"owner":{"own":true,"enemy":true},"cardType":{"hero":true,"minion":true,"player":true},"attackType":{"melee":true,"ranged":true,"hybrid":true}},"availabilityType":{"onPut":true}}],"manaCost":2,"price":100,"remainingNumber":20},{"name":"Tempest","description":"Deal 2 Damage to ALL characters","cardId":"duelyst_Tempest_2","spriteName":"icon_f1_tempest","type":"SPELL","spells":[{"spellId":"deal_2_dmg_everything","action":{"hpChange":-2,"durable":true,"duration":1},"target":{"dimensions":{"row":10,"column":18},"owner":{"own":true,"enemy":true},"cardType":{"hero":true,"minion":true,"player":true},"attackType":{"melee":true,"ranged":true,"hybrid":true}},"availabilityType":{"onPut":true}}],"manaCost":2,"price":100,"remainingNumber":20},{"name":"Tempest","description":"Deal 2 Damage to ALL characters","cardId":"duelyst_Tempest_3","spriteName":"icon_f1_tempest","type":"SPELL","spells":[{"spellId":"deal_2_dmg_everything","action":{"hpChange":-2,"durable":true,"duration":1},"target":{"dimensions":{"row":10,"column":18},"owner":{"own":true,"enemy":true},"cardType":{"hero":true,"minion":true,"player":true},"attackType":{"melee":true,"ranged":true,"hybrid":true}},"availabilityType":{"onPut":true}}],"manaCost":2,"price":100,"remainingNumber":20},{"name":"Void Pulse","description":"Deal 2 damage to the enemy General. Your 
```

Comment:
Was this intended ?

Reply:
No, I think I messed it up.

Reply:
I replaced this file with the master branch